docs/otel.md

```diff
@@ -95,3 +95,90 @@ cargo run --bin trustd api --db-password trustify --auth-disabled --metrics enab
 Access Trustify at [localhost:8080](http://localhost:8080) and analyze the metrics using the [Prometheus UI](http://localhost:9090/)
 
 To view Trustify's metrics in Prometheus, use the following query `{exported_job="trustify"}` and click execute button.
+
+## Diagrams
+
+The diagrams below reflect the current state of both Trustify and [Trustify Helm Charts](https://github.com/trustification/trustify-helm-charts)
+
+### Scenarios
+
+#### Development
+
+> Based on **etc/telemetry/compose.yaml** and **etc/telemetry/compose-tempo.yaml**
+
+```
```

Comment:
```suggestion
```mermaid
```

Reply:
Thanks for catching this : )
First time I'm using Markdown Preview plugin for Neovim, and the plugin just forgot to complain about this

![2025-05-22_05-55](https://github.com/user-attachments/assets/bdbcbd07-a780-46b4-a82d-5af4c924b42d)